src/app/models/AudioRecording.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { generateAudioRecording } from "@test/fakes/AudioRecording";
+import { AudioRecording } from "./AudioRecording";
+
+describe("AudioRecording", () => {
+  let audioRecordingModel: AudioRecording;
+
+  const createModel = (): AudioRecording =>
+    new AudioRecording(generateAudioRecording());
+
+  beforeEach(() => audioRecordingModel = createModel());
+
+  it("should create", () => {
+    expect(audioRecordingModel).toBeInstanceOf(AudioRecording);
+  });
+
+  it("should construct correct batch download route", () => {
+    const expectedUrl = "/audio_recordings/download";
+    const realizedUrl = audioRecordingModel.getBatchDownloadUrl();
+
+    expect(realizedUrl).toEqual(expectedUrl);
+  });
+
+  it("should construct correct analyses download URL", () => {
+    const expectedUrl = `/audio_recordings/${audioRecordingModel.id}/results`;
+    const realizedUrl = audioRecordingModel.getAnalysisResultsUrl();
+
+    expect(realizedUrl).toEqual(expectedUrl);
+  });
```

Comment:
I've completed this by explicitly creating tests for each route.

As this is not ideal, I'll have a look into if I can use some sort of `for in` loop to prevent a lot of duplicate code.